iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ThreeSides.java

```diff
@@ -35,4 +35,22 @@ public static ThreeSides valueOf(int num) {
             default -> throw new PowsyblException("Cannot convert integer value " + num + " to ThreeSides.");
         };
     }
+
+    public Branch.Side toBranchSide() {
+        return switch (this) {
+            case ONE -> Branch.Side.ONE;
+            case TWO -> Branch.Side.TWO;
+            case THREE -> throw new PowsyblException("A Branch has only two sides, ONE and TWO, "
+                                                      + "here it is called with " + this.name());
+        };
+    }
+
+    public ThreeWindingsTransformer.Side toThreeWindingsTransformerSide() {
+        return switch (this) {
+            case ONE -> ThreeWindingsTransformer.Side.ONE;
+            case TWO -> ThreeWindingsTransformer.Side.TWO;
+            case THREE -> ThreeWindingsTransformer.Side.THREE;
+        };
+    }
```

Comment:
Same as for `ThreeSides`, we can introduce 2 attributes, `branchSide` and `threeWindingsTransformerSide`, in the enum to avoid this switches. For `THREE`, we can use `null` for `branchSide` (and throw the exception when this value is encountered in `toBranchSide`).

Reply:
I have a null when I do that: it seems not possible to do both that on ThreeSides and Branch.Side (or ThreeSides and 3WT.Side) because then ThreeSides.ONE need Branch.ONE to be created and Branch.ONE need ThreeSides.ONE to be created (I am not 100% sure this is the problem, but I have a null)

Reply:
We can still do that but only for ThreeSides or only for both Branch.Side and 3WT.Side

Reply:
I have done it one ThreeSides so it is not possible to do it on Branch.Side or 3WT.Side

Reply:
You're right I haven't thought of this cyclic dependency.
Ok.